lib/nexpose/manage.rb

```diff
@@ -43,6 +43,27 @@ def system_information
       end
     end
 
+    # Obtain the version information for each scan engine.
+    # Includes Product, Content, and Java versions.
+    #
+    def engine_versions
+      info = console_command('version engines')
+      versions = []
+      engines = info.split(/\n\n/)
+      engines.each do |eng|
```

Comment:
so maybe after this line we do 
`eng.sub('VERSION INFORMATION',  '')`
and then just let it roll on through and drop the `if` statement? 

Thoughts?